Yoast/wordpress-seo
# Removed redundant markdown plugin

## Summary

This PR can be summarized in the following changelog entry:

* [not-userfacing] Removes redundant markdown plugin for YoastSEO.js

## Relevant technical choices:

* Because the way jetpack works this plugin is no longer relevant. Both in the gutenberg and classic editor the plugin receives content that was already parsed to HTML.

## Test instructions

This PR can be tested by following these steps:

* Install jetpack and set it to development mode by setting `define( 'JETPACK_DEV_DEBUG', true);` in `wp-config.php`.
* Under `Jetpack`> `Settings` > `Writing`, enable `Write posts or pages in plain-text Markdown syntax`.
* Now create a new post or page containing markdown and see to it that the text is still properly analysed the analyses.

## UI changes

## Quality assurance


Fixes #
